bungee/src/main/java/codecrafter47/bungeetablistplus/handler/NewTabOverlayHandler.java

```diff
@@ -299,10 +304,10 @@ public void onServerSwitch(boolean is13OrLater) {
 
         serverPlayerListListed.clear();
         if (serverHeader != null) {
-            serverHeader = EMPTY_JSON_TEXT;
+            serverHeader = EMPTY_TEXT_JSON;
         }
         if (serverFooter != null) {
-            serverFooter = EMPTY_JSON_TEXT;
+            serverFooter = EMPTY_TEXT_JSON;
```

Comment:
Again, not sure if this should be a JSON with an empty `text` property or an empty string 🤔